src/operator/fusion/fused_op.cu

```diff
@@ -253,11 +258,18 @@ std::string FusedOp::GenerateCode(const std::vector<OpReqType> &req,
         CHECK_EQ(outputs[i], 1);
       } else {
         std::string op_name = source->op()->name;
-        if (fusion::slice_ops.find(op_name) != fusion::slice_ops.end()) {
+        bool is_broadcast_op = fusion::broadcast_ops.find(op_name) != fusion::broadcast_ops.end();
```

Comment:
Why define it multiple times? Bring it (and also make `is_slice_op`) outside to the outer loop.